### PR TITLE
#167725508 Signup

### PR DESCRIPTION
**Pull request by wangonya**
## What does this PR do?
- [x] Signup user

### Signup validation
Screenshot 2019-08-13 at 13 21 44
Screenshot 2019-08-13 at 13 20 45

### Successful signup
Screenshot 2019-08-13 at 13 24 02

### CODE REVIEW

**Comment by kburudi** [comment]
@wangonya, I don't think we require password confirmation in the backend. ie, password1 and password2.

**Comment by willeswa** [comment]
> @wangonya, I don't think we require password confirmation in the backend. ie, password1 and password2.

That's right. Password confirmation only checks for misspells since passwords are masked.

**Comment by wangonya** [comment]
@kburudi `django-rest-auth` requires that by default. I've not found a way to disable that option.

File: recipes/authentication/tests/test_signup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import pytest
+
+url = '/rest-auth/registration/'
+test_user = {
+        'username': 'test',
+        'email': 'test@mail.com',
+        'password1': 'testPass1!',
+        'password2': 'testPass1!'
```
Comment (jonathanmusila) [inline recipes/authentication/tests/test_signup.py]: Kindly change the password part on tests too because you have changed the code.

Comment (wangonya) [reply recipes/authentication/tests/test_signup.py]: Done

**Comment by wangonya** [comment]
> @wangonya, I don't think we require password confirmation in the backend. ie, password1 and password2.

@kburudi @willeswa this is fixed.

**Pull request merged**